public/app/features/dashboard-scene/scene/DashboardSceneRenderer.tsx

```diff
@@ -61,6 +63,7 @@ export function DashboardSceneRenderer({ model }: SceneComponentProps<DashboardS
       {!editPanel && (
         <div className={cx(styles.pageContainer, hasControls && styles.pageContainerWithControls)}>
           <NavToolbarActions dashboard={model} />
+          <ScopesDashboards />
```

Comment:
The suggested dashboards list should appear on all pages, not only when viewing a dashboard. The idea behind it is to allow quick navigation to a dashboard from other areas in Grafana.

This is an example of a workflow:
- Navigate to homepage
- Pick a scope
- Expand the suggested dashboards list and navigate to one of the dashboards
- Navigate away to another area in Grafana where scopes make sense (for the time being there ain't one, but just conceptually)
- Do some work in that area, even changing the scope(s)
- Navigate to one of the suggested dashboards

Reply:
@bfmatei currently the scope selector and dashboard list is only rendered on the dashboard page, is this just a current temp thing?

Reply:
Yeah. On the long run we want to expand the usage to other areas such as plugins, explore etc.